<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="widgets">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="idle"/>
    <widget class="QWidget" name="svr_stream">
     <widget class="QLabel" name="svr_stream_lab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Server Stream</string>
      </property>
     </widget>
     <widget class="QLabel" name="svr_stream_lab_port">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port: </string>
      </property>
     </widget>
     <widget class="QLabel" name="svr_stream_lab_ip">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>IP: </string>
      </property>
     </widget>
     <widget class="QLabel" name="svr_stream_lab_audio_file">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Audio File: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="svr_stream_input_audio_file">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="svr_stream_txt_ip">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>_______</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="svr_stream_input_port">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="svr_stream_btn_audio_file">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="svr_stream_btn_start">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="svr_stream_box_clnt_info">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>391</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="svr_stream_lab_clnt_info">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Client Info:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="clnt_stream">
     <widget class="QLabel" name="clnt_stream_lab_port">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Port: </string>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_stream_lab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Client Stream</string>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_stream_lab_ip">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>IP: </string>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_stream_lab_svr_info">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Server Info:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clnt_stream_btn_start">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="clnt_stream_input_port">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="clnt_stream_input_ip">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_stream_lab_curr_playing">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Currently Playing:</string>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_stream_lab_song">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Song: </string>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_stream_lab_artist">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Artist: </string>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_stream_txt_song">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>______</string>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_stream_txt_artist">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>______</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="svr_files">
     <widget class="QLabel" name="svr_files_lab_ip">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>IP: </string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="svr_files_box_activity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>391</width>
        <height>101</height>
       </rect>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QLabel" name="svr_files_lab_client_activity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Client Activity:</string>
      </property>
     </widget>
     <widget class="QLabel" name="svr_files_txt_connected">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>_____</string>
      </property>
     </widget>
     <widget class="QLabel" name="svr_files_lab_connected">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Connected:</string>
      </property>
     </widget>
     <widget class="QLabel" name="svr_files_txt_packets_sent">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>_____</string>
      </property>
     </widget>
     <widget class="QPushButton" name="svr_files_btn_start">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLabel" name="svr_files_lab_packets_recv">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Packets Received: </string>
      </property>
     </widget>
     <widget class="QLabel" name="svr_files_txt_packets_rcv">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>_____</string>
      </property>
     </widget>
     <widget class="QLabel" name="svr_files_lab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Server File Transfer</string>
      </property>
     </widget>
     <widget class="QLabel" name="svr_files_lab_packets_sent">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Packets Sent: </string>
      </property>
     </widget>
     <widget class="QLabel" name="svr_files_lab_port">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Port: </string>
      </property>
     </widget>
     <widget class="QLabel" name="svr_files_txt_ip">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>_____</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="svr_files_txt_port">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="clnt_files">
     <widget class="QLineEdit" name="clnt_files_input_ip">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>29</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="clnt_files_input_rb_download">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>69</y>
        <width>85</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_files_ip">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>29</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_files_lab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>9</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Client File Transfer</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="clnt_files_input_rb_upload">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>69</y>
        <width>85</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="clnt_files_input_port">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>49</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_files_lab_svr_info">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>89</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>File: </string>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_files_lab_port">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>49</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Port: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="clnt_files_input_btn_browse">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_files_lab_type">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>69</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clnt_files_btn_send">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>9</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="clnt_files_input_file">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="clnt_files_box_activity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>391</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_files_lab_activity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Server Activity:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="clnt_voice">
     <widget class="QLabel" name="clnt_voice_lab">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Voice Chat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clnt_voice_btn_call">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>160</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Call</string>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_voice_lab_ip">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>IP: </string>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_voice_lab_port">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Port: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="clnt_voice_txt_ip">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>160</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="clnt_voice_txt_port">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_voice_lab_connect_to">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Connect To:</string>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_voice_lab_user_activity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Your Port:</string>
      </property>
     </widget>
     <widget class="QLabel" name="clnt_voice_lab_your_port">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Port: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="clnt_voice_txt_your_port">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="clnt_voice_btn_accept_call">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Accept Calls</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>391</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="media_slider_progress">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="media_btn_play">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="media_btn_stop">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="media_btn_vol">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/medium-volume.png</normaloff>:/icons/medium-volume.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QSlider" name="media_slider_vol">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="sliderPosition">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="media_lab_txt_curr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="meida_lab_txt_length">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="media_lab_txt_curr_song">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Song</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>409</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServer">
    <property name="title">
     <string>Server</string>
    </property>
    <addaction name="separator"/>
    <addaction name="menu_svr_files"/>
    <addaction name="menu_svr_stream"/>
   </widget>
   <widget class="QMenu" name="menuClient">
    <property name="title">
     <string>Client</string>
    </property>
    <addaction name="menu_clnt_files"/>
    <addaction name="menu_clnt_stream"/>
    <addaction name="menu_clnt_voice"/>
   </widget>
   <addaction name="menuServer"/>
   <addaction name="menuClient"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menu_svr_files">
   <property name="text">
    <string>File Transfer</string>
   </property>
  </action>
  <action name="menu_svr_stream">
   <property name="text">
    <string>Stream</string>
   </property>
  </action>
  <action name="menu_clnt_files">
   <property name="text">
    <string>Upload/Download</string>
   </property>
  </action>
  <action name="menu_clnt_stream">
   <property name="text">
    <string>Stream</string>
   </property>
  </action>
  <action name="menu_clnt_voice">
   <property name="text">
    <string>Voice Call</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
